packages/blade/src/components/Typography/Heading/Heading.tsx

```diff
@@ -0,0 +1,85 @@
+import type { ReactElement } from 'react';
+import type { TextTypes } from '../../../tokens/theme/theme';
+import getPlatformType from '../../../utils/getPlatformType';
+import type { Theme } from '../../BladeProvider';
+import BaseText from '../BaseText';
+import type { BaseTextProps } from '../BaseText/BaseText';
+
+type HeadingCommonProps = {
+  type: TextTypes;
+  children?: React.ReactNode;
```

Comment:
Should we make the childrens required? 

Whats the usecase when any text/heading component will be used without children or ie: be empty?

Reply:
Kept it intentionally, need to fix this for the entire Typography suite.

Reply:
Addressed in https://github.com/razorpay/blade/pull/467